src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -246,6 +274,36 @@ public function isAuthenticated()
     }
 
     /**
+     * Checks if the user is impersonated or not.
+     *
+     * @return bool true if the user is impersonated, false otherwise
+     */
```

Comment:
This phpdoc block and the two next ones can be removed and they don't bring much additional value.